### PR TITLE
Fix integration flutter test yaml

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
_#skip-changelog_

## :bulb: Motivation and Context

> [Invalid workflow file: .github/workflows/flutter_integration_test.yml#L30](https://github.com/getsentry/sentry-dart/actions/runs/5576553131/workflow)
> You have an error in your yaml syntax on line 30

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [ ] I updated the docs if needed
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: .github/workflows/flutter_integration_test.yml [hunk 1]
```diff
@@ -16,72 +16,72 @@ jobs:
...
+          channel: ${{ matrix.sdk }}
+
+      - name: flutter upgrade
+        run: flutter upgrade
+
+      - name: flutter pub get
+        run: flutter pub get
+
+      - name: Gradle cache
+        uses: gradle/gradle-build-action@v2
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply .github/workflows/flutter_integration_test.yml:50]: its a verified action

[the remark this answers is not in the document]
Comment (denrase) [reply .github/workflows/flutter_integration_test.yml:50]: Is there a way to disable the check for individual actions?

**Review by denrase — APPROVED** [review]
Running again, thank you!

File: .github/workflows/flutter_integration_test.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (marandaneto) [reply .github/workflows/flutter_integration_test.yml:50]: I don't think so, it's an org-level action.
It only comments if the line has changed.

**Pull request merged**